bridge/opentracing/internal/mock.go

```diff
@@ -30,6 +30,7 @@ import (
 	"go.opentelemetry.io/otel/bridge/opentracing/migration"
 )
 
+//nolint:revive
```

Comment:
I would like to know what was ignored and why
```suggestion
//nolint:revive // ignoring missing comments for unexported global variables in an internal package
```